popup.js

```diff
@@ -0,0 +1,378 @@
+    type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.`,
+    skills: ['HTML', 'css', 'javaScript', 'Ruby', 'Boostrap'],
+  },
+  {
+
+    class: 'container1',
+    snapshoot: 'Images/Snapshoot4.png',
+    desktop: 'Images/Desktop-img4.png',
+    Tonic: 'Multi-Post Stories',
+    offer: ['CANOPY', 'Back End Dev', '2015'],
+    description: `A daily selection of privately personalized reads; no accounts or
+            sign-ups required.`,
+    descriptionpop1: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
+    It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent`,
+    descriptionpop: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley 
+    of type and scrambled it 1960s. Lorem Ipsum is 
+    simply dummy text of the printing and typesetting industry. 
+    Lorem Ipsum has been the industry's standard dummy text ever 
+    since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of 
+    type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.`,
+    skills: ['HTML', 'css', 'javaScript', 'Ruby', 'Boostrap'],
+  },
+];
+
+const Workcards = document.getElementById('container');
+
+containersworks.forEach((item2) => {
+  Workcards.innerHTML += `
+<div class="${item2.class}">
+        <img
+          class="desktop-image"
+          src="${item2.desktop}"
+          alt="Project Image"
+        />
+        <img
+          class="snapshoot"
+          src="${item2.snapshoot}"
+          alt="Project Image"
+        />
+        <div class="daily">
+          <h3 class="Tonic">${item2.Tonic}</h3>
+          <ul class="offer">
+            <li class="canopy">${item2.offer[0]}</li>
+            <span class="point"></span>
+            <li class="backend">${item2.offer[1]}</li>
+            <span class="point"></span>
+            <li class="year">${item2.offer[2]}</li>
+          </ul>
+            <p class="description">${item2.description}</p>
+          <ul class="skills">
+            <li class="htl"><a href="#"></a>${item2.skills[0]}</li>
+            <li class="css"><a href="#"></a>${item2.skills[1]}</li>
+            <li class="js"><a href="#"></a>${item2.skills[2]}</li>
+          </ul>
+          <button class="seeproject" data-modal-target="#modal1">See Project</button>
+        </div>
+      </div>
```

Comment:
- [ ] [OPTIONAL] Please since we are having the desktop version of the application, I suggest to consider making the works card adaptive to the different screen sizes of the mobile version i.e 768px and below. This will make your portfolio more professional and user-friendly.

Your design at 600px
![image](https://user-images.githubusercontent.com/99394374/232922113-2ba46161-ddc6-4273-b730-ed2c4ee1d016.png)